### DISCUSSION TITLE
GLPI-Agent 1.6.1

### DISCUSSION THREAD

**Discussion by g-bougard** [glpi-project/glpi-agent#539] [Announcements]
### Announcement

See official announcement from the GLPI-Project blog:
https://glpi-project.org/glpi-agent-1-6-1/

**Comment by edwinsyn** [comment]
Hi @g-bougard, I got this error when installing a fresh GLPI agent on DEB based Linux using the .pl script. Do you know if I did something wrong from my end?

`Can't locate object method "open_os_file" via package "self" (perhaps you forgot to load "self"?) at glpi-agent-1.6.1-linux-installer.pl line 589.`

**Reply by g-bougard** [reply]
Hi @edwinsyn 
please, don't ask such question here, this is not the right place.
Also if you have searched closed issues, you would have found this issue has still been fixed. So you can just use a nightly build. A new release will be published in a week or two.

**Reply by alessandrojbr** [reply]
Hi @edwinsyn 

See the solution. https://github.com/glpi-project/glpi-agent/commit/854ee13d869404f4ee318d06ed4a627ea9a0283e
